dlq_processor/cloudwatchlogs_lambda.js

```diff
@@ -137,7 +137,7 @@ exports.processLogs = function (env, eventAwslogsData, errorHandler) {
             };
             // removing headers with 'none'
             for (var key in headers) {
-                if (headers[key] === 'none')
+                if (!headers[key] || (headers[key].toLowerCase() === 'none'))
```

Comment:
Does this still apply? I don't see anywhere we set 'none'.

Reply:
yes if the customer sets it none then it won't be replace by loggroup/logstream and source name and host won't be sent in headers